pkgs/by-name/re/resnap/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  pkgs,
+  fetchFromGitHub,
+  writeShellApplication,
+}:
+writeShellApplication {
+  name = "reSnap";
+
+  runtimeInputs = with pkgs; [
+    ffmpeg
+    feh
+    imagemagick_light
+    lz4
+  ];
```

Don't use `pkgs` inside the derivation. Instead, add the dependencies to the inputs.

```suggestion
  runtimeInputs = [
    ffmpeg
    feh
    imagemagick_light
    lz4
  ];
```